lib/studygroup_web/live/page_live.html.leex

```diff
@@ -1,10 +1,10 @@
 <section class="container mx-auto py-24 px-4">
   <div class="w-full md:w-11/12 xl:w-8/12 mx-auto text-left md:text-center">
     <h1 class="text-4xl md:text-5xl text-gray-900 leading-tight mb-3 font-bold md:font-extrabold">
-      Thank you for using Boilerplate Generator
+      Greetings! <br> This is our Elixir Learning field
     </h1>
     <p class="text-lg md:text-xl text-gray-600 mb-6">
-      You are now ready to start or continue your journey to greatness.
+      github url - rmilushev/studygroup
```

Comment:
I suggest to add clarification text instead of GitHub link, like: `If you want to join the study group, please, follow us...`